### PR TITLE
bugfix:Fixes/stop services

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

 Fixes an error in the implementation of ``stop_services`` in ``services/web/server/src/simcore_service_webserver/director/director_api.py`` prevents from stopping services.

#### Other minor fixes
   - api-server: 
        - FIX: typo in ``LOG_LEVEL`` environment variable prevented setting log level
        - FIX: missing coverage argument in ci bash script
   - FIX: missing namespace in ``test_projects``
   - FIX: loop fixture missing in ``test_aiopg_utils`` would produce constant ``FileNotFoundError`` messages upon cleanup
   

## How to test

Added a new test ``services/web/server/tests/unit/isolated/test_director_api.py`` that mocks director service API and
runs all functions in ``services/web/server/src/simcore_service_webserver/director/director_api.py`` in a workflow

```
cd osparc-simcore
make devenv
source .venv/bin/activate
cd services/web/server
make install-dev
pytest tests/unit/isolated/test_director_api.py
```

### CODE REVIEW

File: packages/pytest-simcore/src/pytest_simcore/schemas.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def node_meta_schema(node_meta_schema_file: Path) -> Dict:
 
 
 @pytest.fixture(scope="session")
-def json_schema_dict(common_schemas_specs_dir: Path) -> Callable:
+def json_schema_dict(common_schemas_specs_dir: Path) -> Iterable[Callable]:
```
Comment (sanderegg) [inline packages/pytest-simcore/src/pytest_simcore/schemas.py:36]: is that not more like:
```python
Callable[[str], Dict[str,Any]]
```?

File: packages/pytest-simcore/src/pytest_simcore/schemas.py [hunk 2]
```diff
@@ -55,13 +55,16 @@ def json_faker_script(script_dir: Path) -> Path:
 @pytest.fixture(scope="session")
 def random_json_from_schema(
     json_faker_script: Path, tmp_path_factory: Path
-) -> Callable:
-    def _generator(schema: Dict) -> Dict:
+) -> Iterable[Callable]:
```
Comment (sanderegg) [inline packages/pytest-simcore/src/pytest_simcore/schemas.py:58]:
```suggestion
) -> Callable[[str], Dict[str,Any]]:
```

File: services/web/server/tests/unit/isolated/test_director_api.py [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+        )
+
+    async with ClientSession() as session:
+        async with session.get(
+            # "http://director:8000/v0/running_interactive_services"
+            "http://director:8001/v0/running_interactive_services?project_id=dc7d6847-a3b7-4905-bbfb-777e3bd433c8&user_id=1"
+        ) as resp:
+            assert resp.status == 200
+            print(await resp.json())
+
```
Comment (sanderegg) [inline services/web/server/tests/unit/isolated/test_director_api.py:334]: really want to keep this?

**Review by sanderegg — APPROVED** [review]
beautiful! some minor comments but seems to work nicely!

**Review by mguidon — APPROVED** [review]

File: packages/pytest-simcore/src/pytest_simcore/schemas.py (hunk 2 above)
Comment (pcrespov) [reply packages/pytest-simcore/src/pytest_simcore/schemas.py]: added  (needed to add also the import)

File: services/web/server/tests/unit/isolated/test_director_api.py (hunk 3 above)
Comment (pcrespov) [reply services/web/server/tests/unit/isolated/test_director_api.py]: removed

**Pull request merged**